src/components/Tooltip/components/TooltipOverlay/TooltipOverlay.tsx

```diff
@@ -46,12 +53,13 @@ export class TooltipOverlay extends React.PureComponent<
   ) => {
     const {measuring, desiredHeight, positioning} = overlayDetails;
 
-    const {id, children, light} = this.props;
+    const {id, children, light, preventInteraction} = this.props;
 
     const containerClassName = classNames(
       styles.Tooltip,
       light && styles.light,
       measuring && styles.measuring,
+      preventInteraction && styles.preventInteraction,
```

Comment:
There is no class that you have authored within `Tooltip.scss` for `preventInteraction`. This class only exists in `PositionedOverlay`, and should only be needed in that one location.